playbooks/archive_artifacts.yml

```diff
@@ -0,0 +1,77 @@
+- hosts: "{{ hosts }}"
```

Comment:
It's use doesn't shadow anything, not sure it's really a problem but happy to change.